### PR TITLE
Nicer (& fixed up) help texts for space views

### PR DESCRIPTION
**Pull request by Wumpf**
Introduced a little utility for this purpose. Not very well-rounded, but a good starting point I like to think :)

Help texts for different space views:

3D:
image

2D:
image

Bar chart:
image

Time series:
image

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)

### CODE REVIEW

**Comment by Wumpf** [comment]
on the 3D it says `holding alt` instead of `holding option` despite me being on a mac.
https://github.com/emilk/egui/pull/2981 fixes this!

File: crates/re_viewer/src/ui/view_spatial/ui_3d.rs [hunk 1]
```diff
@@ -238,18 +238,55 @@ fn find_camera(space_cameras: &[SpaceCamera3D], needle: &EntityPath) -> Option<E
...
-    \n\
-    Double-click on empty space to reset the view.";
+pub fn help_text(re_ui: &re_ui::ReUi) -> egui::WidgetText {
+    let pan_mouse = egui::PointerButton::Secondary;
+    let roll_mouse = egui::PointerButton::Middle;
+    let roll_mouse_alt = egui::PointerButton::Primary;
+    let roll_modifier = egui::Modifiers::ALT;
+    let speed_up = egui::Modifiers::SHIFT;
+    let slow_down = egui::Modifiers::CTRL;
+    let restore_key = egui::Key::Escape;
```
Comment (jleibs) [inline crates/re_viewer/src/ui/view_spatial/ui_3d.rs:248]: Would be nice if these semantic constants were shared with the values used here:
https://github.com/rerun-io/rerun/blob/78c99361cb52cafe2c8894ab9ff9e58cef1763e5/crates/re_viewer/src/ui/view_spatial/eye.rs#L298-L317

Comment (Wumpf) [reply crates/re_viewer/src/ui/view_spatial/ui_3d.rs:248]: was too lazy to go that extra mile, but yeah at least some of those are pretty easy. Let's put them somewhere central!

Comment (emilk) [reply crates/re_viewer/src/ui/view_spatial/ui_3d.rs:248]: that would be nice

**Review by emilk — APPROVED** [review]

File: crates/re_viewer/src/ui/view_spatial/ui_3d.rs (hunk 1 above)
Comment (Wumpf) [reply crates/re_viewer/src/ui/view_spatial/ui_3d.rs]: done that now. Sadly lots of controls are hidden inside other egui constructs like scrollview or plot

File: crates/re_viewer/src/ui/spaceview_controls.rs [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                ..contains_query
+            },
+        );
+    }
+    if (mac_cmd || command) && (contains_query.mac_cmd || contains_query.command) {
+        return modifier_contains(
+            modifiers,
+            egui::Modifiers {
+                mac_cmd: false,
+                ctrl: false,
```
Comment (Wumpf) [inline crates/re_viewer/src/ui/spaceview_controls.rs:94]: wrong! Fixed  in https://github.com/emilk/egui/pull/2989

**Pull request merged**